src/context/AuthContext.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react'
+
+const AuthContext = React.createContext({
+   jwt: '',
+   isLoggedIn: false,
+   login: (token) => {},
+   logout: () => {},
+})
```

Comment:
You don't have to provide initial values, it gets messy when you have many properties in the context :D you can just do this

```suggestion
const AuthContext = React.createContext(undefined)
```